.github/workflows/run-local-tests.yml

```
@@ -0,0 +1,21 @@
+      fail-fast: false
+      matrix:
+        test_target: [unit-test, sqlt-test]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup Test Environment
+        uses: ./.github/actions/setup-test-env
+
+      - name: Run local tests (USE_SQLITE=true)
+        run: make ${{ matrix.test_target }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)